hbase-shaded-jaeger/pom.xml

```diff
@@ -0,0 +1,176 @@
+<?xml version="1.0"?>
+<project xmlns="https://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="https://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="https://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <!--
+  /**
+   * Licensed to the Apache Software Foundation (ASF) under one
+   * or more contributor license agreements.  See the NOTICE file
+   * distributed with this work for additional information
+   * regarding copyright ownership.  The ASF licenses this file
+   * to you under the Apache License, Version 2.0 (the
+   * "License"); you may not use this file except in compliance
+   * with the License.  You may obtain a copy of the License at
+   *
+   *     http://www.apache.org/licenses/LICENSE-2.0
+   *
+   * Unless required by applicable law or agreed to in writing, software
+   * distributed under the License is distributed on an "AS IS" BASIS,
+   * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   * See the License for the specific language governing permissions and
+   * limitations under the License.
+   */
+  -->
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>hbase-thirdparty</artifactId>
+        <groupId>org.apache.hbase.thirdparty</groupId>
+        <version>3.2.1-SNAPSHOT</version>
```

Comment:
Since this adds a new dependency and artifact we should bump the hbase-thirdparty minor version (sorry we don't have guidelines written up for this)

Reply:
thanks! i looked at the git history and jiras but i don't get a good sense of when we bump major/minor/maint version number. Is there a DISCUSS thread I missed?

Reply:
Bumped the version to 3.3.0-SNAPSHOT. I believe I should do this for all shaded thirdparty libraries.